### ISSUE TITLE
prometheus metrics exposed

### ISSUE THREAD

**Issue by tluimes** [sse-secure-systems/connaisseur#511]
**Describe the feature**
<A clear and concise description of the feature you envision.>
A `/metrics` endpoint which exposes http request duration, and metrics for admission decisions.

***Metrics***
```
# HELP http_request_duration_seconds Flask HTTP request duration in seconds
# TYPE http_request_duration_seconds histogram
```
```
# HELP mutate_requests_total Total number of mutate requests
# TYPE mutate_requests_total counter
```

**Optional: Is your feature request related to a problem? Please describe.**
<A clear and concise description of what the problem is.>
We would like to know if of latency is introduced to the image pull flow because of changes in the code or potential resource contention. Also, it would be useful to know if there is a high `allowed = False` response because of either misconfiguration, code logic, or downstream issues.

**Optional: Implementation ideas**
<Maybe you already have an implementation and alternatives in mind. Please share your thoughts.>
Could be implemented using [prometheus-flask-exporter](https://pypi.org/project/prometheus-flask-exporter/) and adding a custom metric based on the response payload.

**Optional: Additional context**
<Add any other context or screenshots about the feature request here.>

**Comment by xopham** [comment]
@tluimes great feature proposal! Totally support it :+1:

**Comment by xopham** [comment]
merged as part of the `v2.5.0` release! :rocket:
